sysutils/libvirt/Portfile

```diff
@@ -50,97 +48,77 @@ depends_lib         port:curl \
 # error: You need at least XCode Clang v5.1 to compile QEMU
 compiler.blacklist-append apple-gcc* gcc-3.3 *gcc-4.* macports-clang-3.3 {clang < 503}
 
+patchfiles-append   patch-meson.diff
```

Comment:
Instead of patching, `${frameworks_dir}/Python.framework/Versions/3.8/bin` could be prepended to PATH. That would make it easier to change the python version in the future, since only the Portfile (and not also the patchfile) would need to be changed. By abstracting 3.8 into a variable and using it everywhere, it could be reduced to a single portfile line that would need to be changed when switching the python version, ensuring that we don't forget to switch something.

Reply:
This helps a lot. I looked at the failed github check and was not sure how to solve the missing python3 link and did not like adding another patch to fix meson.build for that as well. Will do both.